homeassistant/components/rachio/__init__.py

```diff
@@ -83,7 +83,7 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     if not person.controllers and not person.base_stations:
         _LOGGER.error("No Rachio devices found in account %s", person.username)
         return False
-    _LOGGER.warning(
+    _LOGGER.debug(
```

Comment:
I wonder if we should fallback to polling if the webhook isn't working

Reply:
Probably not a bad idea, I'd imagine it would be a decent amount of work since everything on the controller side only listens for webhooks right now. I can look and see what that would take at some point.